docs/app/pages/home/backend-bundles-section/backend-bundles-section.component.html

```diff
@@ -0,0 +1,248 @@
+<ngx-landing-section-title>
+  Backend Bundles
+</ngx-landing-section-title>
+
+<div class="section-container">
+  <ul class="features">
+    <li class="features__item">
+      <h3 class="features__title title">Convenient</h3>
+      <p class="features__description">Complete pack of well known Angular based ngx-admin template, integrated with Backend Solution. Finally, you can get fully backend integrated solution out of the box. Flexibility of Ngx Admin, the variety of Nebular features and convenience of integrated Backend in one pack.
+      </p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Functional</h3>
+      <p class="features__description">Functional Backend Solution with Designed and Implemented Services and Data Flow. Deploy it as ready to use solution for a particular case, or give it to your development team to incrementally add functionality for the bigger system. it gives you a significant boost on start and solid development structure.</p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Efficient</h3>
+      <p class="features__description">Save more than $21,000 using Backend Bundle. According to our research usage of a ready Backend Bundle optimizes 300 hours of development time. Taking the average hourly rate of a full stack developer in the US - $70.</p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Ready to use</h3>
+      <p class="features__description">We prepared this backend pack as development basement which lets your team concentrate on business logic and data models. Now .NET and .NET Core are available as frameworks of choice. More are in progress…</p>
+    </li>
+  </ul>
+
+  <picture class="bundle-scheme">
+    <source type="image/webp" srcset="/assets/img/bundle-scheme@1x.webp 1x, /assets/img/bundle-scheme@2x.webp 2x">
+    <img class="bundle-scheme__image" src="/assets/img/bundle-scheme@1x.png" srcset="/assets/img/bundle-scheme@2x.png 2x" alt="Backend bundles">
+  </picture>
+
+  <div class="package-switcher">
+    <div [(ngModel)]="licenseType" ngbRadioGroup
+         class="package-switcher__btn-group btn-group btn-group-toggle btn-outline-toggle-group btn-group-full-width btn-toggle-radio-group">
+      <label ngbButtonLabel class="package-switcher__btn btn btn-outline-secondary">
+        <input ngbButton type="radio" value="Personal"> Personal
```

Comment:
It's better to extract `Personal` and `Commercial` strings into component properties and use it across the component.
```suggestion
        <input ngbButton type="radio" [value]="personalLicense"> {{ personalLicense }}
```
and in `.ts` file:
```ts
class BackendBundlesSectionComponent {
  ...
  personalLicense = 'Personal';
  commercialLicense = 'Commercial';
  ...
}
```
Since this text is a subject to change, you'll need to find all its occurrences and replace with the new value if it change. Search and replace of strings is error-prone.